packages/cactus-plugin-keychain-aws-sm/package.json

```diff
@@ -0,0 +1,103 @@
+{
+  "name": "@hyperledger/cactus-plugin-keychain-aws-sm",
+  "version": "0.4.1",
```

Comment:
@jagpreetsinghsasan Version will have to be updated as well because we have a release in the queue so keep an eye out for this as well (I know for a fact that the release will be issued before the merging of this PR)

Reply:
Do we know what's the next release, 0.4.2 or 0.5.0? (Like is it planned yet)

Reply:
@jagpreetsinghsasan It's going to be `0.5.0-alpha.0` (see https://github.com/hyperledger/cactus/pull/927 )

You can't update it just yet though because then there will be other problems, but once #927 is merged, then it will be possible (and necessary at that point).